master/reference/felix/configuration.md

```diff
@@ -117,40 +117,18 @@ The Kubernetes API datastore driver reads its configuration from Kubernetes-prov
 
 #### Felix-Typha TLS configuration
 
-{% include {{page.version}}/felix-typha-tls-intro.md %}
-
-To use TLS, each Felix instance must have a certificate and key pair signed by
-a trusted CA.  Felix then uses TLS when connecting to Typha, and requires Typha
-to present a certificate that is signed by a trusted CA and
-has an expected identity in its Common Name or URI SAN field.  Either
-`TyphaCN` or `TyphaURISAN` must be configured, and Felix will check the
-presented certificate accordingly.
-
--  For a [SPIFFE](https://github.com/spiffe/spiffe)-compliant deployment you
-   should configure `TyphaURISAN` with a [SPIFFE
-   Identity](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#2-spiffe-identity)
-   and provision Typha certificates with the same identity in their URI SAN
-   field.
-
--  Alternatively you can configure `TyphaCN` and provision Typha certificates
-   with that `TyphaCN` value in their Common Name field.
-
-If both those parameters are set, a Typha certificate only has to match one of
-them; this is intended for when a deployment is migrating from using Common
-Name to using a URI SAN, to express identity.
-
 | Configuration parameter | Environment variable   | Description | Schema |
 | ----------------------- | ---------------------- | ----------- | ------ |
-| `TyphaCAFile`           | `FELIX_TYPHACAFILE`    | The full path to the certificate file for the Certificate Authorities that Felix trusts for Felix-Typha communications. | string |
-| `TyphaCertFile`         | `FELIX_TYPHACERTFILE`  | The full path to the certificate file for this Felix instance to use to connect to Typha. | string |
-| `TyphaCN`               | `FELIX_TYPHACN`        | If set, the Common Name that Typha's certificate must have. [Default: not set] | string |
-| `TyphaKeyFile`          | `FELIX_TYPHAKEYFILE`   | The full path to the private key file for this Felix instance to use to connect to Typha. | string |
-| `TyphaURISAN`           | `FELIX_TYPHAURISAN`    | If set, a URI SAN that Typha's certificate must have. [Default: not set] | string |
+| `TyphaCAFile`           | `FELIX_TYPHACAFILE`    | Use  this parameter to supply Felix with the path to the file containing the root certificate of the CA that issued the Typha server certificate. Configures Felix to trust the signature on the certificates provided by the Typha server. Example: `/etc/felix/ca.pem` | string |
```

Comment:
I think the previous text was better in several respects:
- "Use this parameter to supply Felix with" is superfluous.
- In general the Felix deployment may not know who issued the Typha certificate.  In reference terms, the point for Felix is which certificate authorities it trusts.
- A CA file can contain multiple certificate authorities, and this is important in practice for some certificate rotation scenarios.  So the plural here is important.
- "trust the signature" sounds wrong to me.  The concept is that the signature indicates a particular signer (certificate authority), and one trusts (or not) the signer.

Reply:
@neiljerram If Typha presents a certificate signed by someone other than that specified in `TyphaCAFile`, will Felix reject the connection?